assets/js/Navbar.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from "react";
+
+export default function Navbar({ user }) {
+  const [toggleMenu, setToggleMenu] = useState(false); // Initialize the menu as closed
+
+  const toggleButton = () => {
+    setToggleMenu(!toggleMenu); // Toggle the menu
+  };
+
+  return (
+    <nav className="bg-blue-500 p-4 fixed top-0 w-full z-10 flex">
+      <h1
+        className={`text-white text-2xl font-semibold ${
+          !toggleMenu ? "self-center" : "self-start"
+        }`}
+      >
+        Criticos
+      </h1>
+      <div className="container mx-auto relative z-10 flex justify-end items-center">
+        {/* Show the navigation items in a dropdown menu when the menu is toggled */}
+        <div
+          className={`md:hidden w-full flex flex-col transition-max-h ${
+            toggleMenu ? "max-h-screen" : "max-h-0 hidden" // Show or hide the dropdown menu based on the toggle state
+          }`}
+        >
+          <a href="#" className="text-white p-2 self-center">
+            Home
+          </a>
+          <a href="#" className="text-white p-2 self-center">
+            Books
+          </a>
+          <a href="#" className="text-white p-2 self-center">
+            Reviews
+          </a>
+          {!user ? (
+            <>
+              <a href="/users/settings" className="text-white p-2 self-center">
+                Settings
+              </a>
+              <a
+                href="/users/log_out"
+                method="delete"
+                className="text-white p-2 self-center"
+              >
+                Log out
+              </a>
+            </>
+          ) : (
+            <>
+              <a href="/users/log_in" className="text-white p-2 self-center">
```

Comment:
Needs to call the login API. Also has to handle what happens if it succeeds or fails